ci.nix

```diff
@@ -11,6 +11,7 @@
   nixpkgsVersions = {
     "R2003" = "nixpkgs-2003";
     "R2009" = "nixpkgs-2009";
+    "R2105" = "nixpkgs-2105";
```

Comment:
I think our policy is to support the last two stables. So let's ditch 20.03.

Then there's a bunch of conditional logic in this file about whether we build some stuff on various nixpkgs version. We should check all of that.

Finally, if we drop 20.03 we can potentially drop some patches etc, we should check.

Reply:
Well, it's going to bulk up our build matrix a lot, so it's good to prune it at the same time! I think most of the conditional logic is in here, but we may have to hunt for other bits too.